### PR TITLE
Feature react frontend jony

### PR DESCRIPTION
**Pull request by J0nki**
Added a React/Typescript FE project.
Added models for BlogPost and Profile that corresponds with the models BE.
Fetching data from BE to simple lists in FE.

### CODE REVIEW

File: sharebookfrontend/src/assets/logo.png
Comment (stenholmjonathan) [inline sharebookfrontend/src/assets/logo.png:1]: remove this please

File: sharebookfrontend/src/assets/react.svg
Comment (stenholmjonathan) [inline sharebookfrontend/src/assets/react.svg:1]: remove this please

File: sharebookfrontend/public/vite.svg
Comment (stenholmjonathan) [inline sharebookfrontend/public/vite.svg:1]: remove this please

File: sharebookfrontend/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```
Comment (stenholmjonathan) [inline sharebookfrontend/index.html:5]: remove the line for the image

File: sharebookfrontend/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```
Comment (stenholmjonathan) [inline sharebookfrontend/index.html:7]: remove title, or change it

File: sharebookfrontend/src/assets/logo.png
Comment (J0nki) [reply sharebookfrontend/src/assets/logo.png:1]: We will keep this

**Review by stenholmjonathan — APPROVED** [review]

**Pull request merged**